### PR TITLE
Front page updates

### PR DESCRIPTION
**Pull request by graeme-a-stewart**
Add future event announcements for HSF workshops in the next few months
Change the centre bar to focus on the AE2 workshop

### CODE REVIEW

File: announcements/_posts/2022-04-13-pyhep2022.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: PyHEP 2022 "Python in HEP" (virtual) Workshop, 12-16 September 2022
+author: Eduardo Rodrigues
+layout: newsletter
+symbol: glyphicon-calendar
+link: https://indico.cern.ch/e/pyhep2021
```
Comment (eduardo-rodrigues) [inline announcements/_posts/2022-04-13-pyhep2022.md:6]:
```suggestion
link: https://indico.cern.ch/e/PyHEP2022
```

**Review by eduardo-rodrigues — APPROVED** [review]

File: announcements/_posts/2022-04-13-pyhep2022.md (hunk 1 above)
Comment (eduardo-rodrigues) [reply announcements/_posts/2022-04-13-pyhep2022.md:6]: Ah ha, just to make sure I read :D.

Comment (graeme-a-stewart) [reply announcements/_posts/2022-04-13-pyhep2022.md:6]: Ah, but now I realise that we don't have a URL yet - shall we create a stub agenda to be able to link to?

Comment (eduardo-rodrigues) [reply announcements/_posts/2022-04-13-pyhep2022.md:6]: Let me do it. It is on my list anyway ... Give me a sec ...

Comment (eduardo-rodrigues) [reply announcements/_posts/2022-04-13-pyhep2022.md:6]: The basic Indico and short link now exist :-). You're good to go.

**Pull request merged**